packages/diracx-web/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "diracx-web",
+  "version": "0.1.0-a1",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build && rm -rf ../../out && mv out ../..",
```

Do you actually need to `rm` `../../out`?

 ```suggestion
    "build": "next build && mv out ../..",
```

---

If the folder isn't removed `mv` fails with the error : `mv: cannot move 'out' to '../../out': Directory not empty`.
I guess the command would work if we used `out/*` but deleted files between builds would remain in the directory.